apps/estuary/lib/estuary/message_handler.ex

```diff
@@ -2,40 +2,25 @@ defmodule Estuary.MessageHandler do
   @moduledoc """
   Estuary.MessageHandler reads an event from the event stream and persists it.
   """
-  alias Estuary.EventTable
   use Elsa.Consumer.MessageHandler
-  require Logger
+  alias Estuary.DataWriter
 
   def handle_messages(messages) do
-    Enum.each(messages, fn message -> process_message(message) end)
+    messages
+    |> Enum.each(fn message ->
+      message.value
+      |> Jason.decode!()
+      |> DataWriter.write()
```

Comment:
have we verified that DataWriter will batch when it writes into Presto for you?
per Matt's previous updates, it sounds like we are expecting this pipeline refactor to solve that problem of estuary only writing a single record at a time so unless DataWriter is doing that for you out of the box, this isn't going to allow anything to get deployed to prod.

Reply:
This guy is still writing only 1 message at a time to presto, despite handling/getting batches of messages from elsa. The next card we play will be WRITING those messages as a batch to presto.